source/Sashimi/AzureWebAppHealthCheckActionHandler.cs

```diff
@@ -26,4 +26,26 @@ public IActionHandlerResult Execute(IActionHandlerContext context, ITaskLog task
                           .Execute(taskLog);
         }
     }
+
+    class AzureWebAppDiscoveryActionHandler : IActionHandler
+    {
+        static readonly CalamariFlavour CalamariAzureAppService = new CalamariFlavour("Calamari.AzureAppService");
+
+        public string Id => SpecialVariables.Action.Azure.WebAppHealthCheckActionTypeName;
+        public string Name => "Discover Azure Web Apps";
+        public string Description => "Discover available Azure Web Apps.";
+        public string? Keywords => null;
+        public bool ShowInStepTemplatePickerUI => false;
+        public bool WhenInAChildStepRunInTheContextOfTheTargetMachine => false;
+        public bool CanRunOnDeploymentTarget => false;
+        public ActionHandlerCategory[] Categories => new[] { ActionHandlerCategory.BuiltInStep, ActionHandlerCategory.Azure };
+
+        public IActionHandlerResult Execute(IActionHandlerContext context, ITaskLog taskLog)
+        {
+            taskLog.Info($"Yo I'm finding some web apps here using context '{context.Variables.Get("Octopus.TargetDiscovery.Context")}'");
```

Comment:
I imagine we want to remove this (we won't say who put it there in the first place) 😆

Reply:
Done!

Reply:
Done.